server/app.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import blogsRouter from './routes/blogs';
+import mongoose from 'mongoose';
+
+// MongoDB Connection
+const dbURI = "mongodb+srv://faustin:faustin@learnnode.e3lxu.mongodb.net/nodeDB?retryWrites=true&w=majority";
```

Comment:
For security purposes, I think it's better to put your dbURI in or your MongoDB username and password in the .env file.